languages/LinkEdit.groovy

```diff
@@ -30,10 +30,17 @@ sortedList.each { buildFile ->
 	buildUtils.copySourceFiles(buildFile, props.linkedit_srcPDS, null, null, null)
 
 	// create mvs commands
-	String rules = props.getFileProperty('linkedit_resolutionRules', buildFile)
-	DependencyResolver dependencyResolver = buildUtils.createDependencyResolver(buildFile, rules)
-	LogicalFile logicalFile = dependencyResolver.getLogicalFile()
+	LogicalFile logicalFile
+	if (props.useSearchConfiguration && props.useSearchConfiguration.toBoolean()) { // use new SearchPathDependencyResolver
+		logicalFile = SearchPathDependencyResolver.getLogicalFile(buildFile,props.workspace)
```

Comment:
Is SearchPathDependencyResolver exits in DBB < 1.1.2 ? If no you may have an exception.

Reply:
I have tested this with 1.0.9 . No Exception.